.github/workflows/config-template.yaml.j2

```diff
@@ -50,13 +60,39 @@ hub:
       scope:
         - read:user
         - read:org
-    JupyterHub:
-      authenticator_class: github
+  {%- elif authenticator_class == 'dummy' %}
+    DummyAuthenticator:
+      password: "{{ auth_password }}"
+  {%- endif %}
+{%- endif %}
+
   db:
     pvc:
+{%- if RELEASE == 'aiidalab-marketplace' %}
+      storageClassName: do-block-storage
+{%- else %}
       storageClassName: gp2
+{%- endif %}
+
   extraConfig:
     myConfig: |
+{%- if RELEASE == 'aiidalab-marketplace' %}
+      {% macro mpoath() %}{% include "./marketplace/oauthenticator.py" %}{% endmacro %}
```

Comment:
What does `mpoath()` do? I can't find it in the Jinja2 [documentation](https://jinja.palletsprojects.com/en/2.11.x/search/?q=mpoath).

Maybe you can add a comment explaining this a little bit?